### DISCUSSION TITLE
Fetching UK Only, or Anywhere jobs

### DISCUSSION THREAD

**Discussion by brettdewoody** [remotive-com/remote-jobs-api#4] [General]
Hi @adriencombourieu, is it possible to fetch jobs via the API by `candidate_required_location` - for "UK Only" or "Anywhere" jobs?

I've tried the search param to search for "UK Only" but it returns 0 results, perhaps because that string isn't in the title or description.

**Comment by adriencombourieu** [comment]
Hi @brettdewoody,

Good question there!
We do not provide filters on location with the free/public API.
We do provide some location filters in a paid version of this API though. 
Reach out if you want to know more!

**Reply by brettdewoody** [reply]
That's great to know. I'll hold off on the paid API for now as I'm just getting started.

Is the paid API more for businesses wanting to post jobs?

**Reply by adriencombourieu** [reply]
We do have an API for businesses wanting to post jobs, but it's another API.

This API here is to list our current job openings. The idea is that it is free for individual developers, students, open source project, non profit community etc... which should always redirect traffic to remotive.io

We provide paid version of this API in some cases such as:
- specific requirements/filtering
- for professional job boards / niche community
